war/src/main/less/base/style.less

```diff
@@ -1756,6 +1833,12 @@ body.no-sticker #bottom-sticker {
   fill: currentColor;
 }
 
+/* -------------------------------------- */
+
+.tr {
```

Comment:
Another name without table implications would be better IMO. Something like _row-top-distanced_

Reply:
See https://github.com/jenkinsci/jenkins/pull/3895#discussion_r448889010